### ISSUE TITLE
Enhancement :Verse Level Navigation in Reading plan

### ISSUE THREAD

**Issue by AnnCherianA** [Bridgeconn/VachanAppReact#112]
: Currently in the reading plan, on selecting a long chapter that mentions the verses such as Psalms 119:145-176, it navigates to the 1st verse of the chapter. 

Enhancement It  would be nice if we could  enable navigation to the 145th verse , when the user selects the  reading plan link, instead of displaying the 1st verse onwards.

 Comment from @samuelanthrayose : -29 June 2021 -As of now, it is not possible to go to a particular verse from the Bible Reading plan. This feature is not built-in. We can only navigate to the book/chapter.  But sometime later,we may need to add the verse level navigation as well.

**Comment by neetuy** [comment]
As discussed in meeting for now we have decided to make bible reading plan simple, later we can add this feature.

**Comment by samuelanthrayose** [comment]
Yes, verse level-navigation is not available as of now. So we could add verse level-navigation functionality during the next phase of development.

**Comment by AnnCherianA** [comment]
@neetuy Linking the below issue as this is related to Verse selector  
https://github.com/Bridgeconn/VachanAppReact/issues/150

**Comment by AnnCherianA** [comment]
@samuelanthrayose we now have verse level navigation implemented in 1.3.2 in such a way that selecting a verse would navigate to a page where the verse is displayed along with other verses. Please check if this is fine.
Overall, the feature looks ok

**Comment by AnnCherianA** [comment]
@neetuy please move along the board to QA (in progress ->QA)  so that we can verify and close
